FND/DatesAndTimes/BusinessDates.rdf

```diff
@@ -135,26 +136,18 @@ They are modularized this way to minimize the ontological committments that are
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>business recurrence interval</rdfs:label>
-		<skos:definition>a recurrence interval that is specified using a business recurrence interval convention</skos:definition>
+		<skos:definition>recurrence interval that is specified using a business recurrence interval convention</skos:definition>
```

Comment:
Outside scope of this PR, but this is a terrible class name for this definition.

Reply:
Created #899